package.json

```diff
@@ -55,7 +55,7 @@
     "@babel/code-frame": "7.24.2",
     "@gemini-testing/commander": "2.15.4",
     "@jspm/core": "2.0.1",
-    "@puppeteer/browsers": "2.4.0",
+    "@puppeteer/browsers": "2.7.1",
```

Comment:
This includes commit https://github.com/puppeteer/puppeteer/commit/d247a7fcd4fc01cf0c53d733632a1e9687b8004d
This commit is essential for downloading firefox@135+